packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -121,10 +118,17 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
   ) {
     let result;
     /** [SXA] it needs for deleting dynamics placeholder when we set him number(props.name) of container.
-    from backend side we get common name of placeholder is called 'container-{*}' where '{*}' marker for replacing **/
-    if (rendering && rendering.placeholders && rendering.placeholders[PREFIX_PLACEHOLDER]) {
-      rendering.placeholders[name] = rendering.placeholders[PREFIX_PLACEHOLDER];
-      delete rendering.placeholders[PREFIX_PLACEHOLDER];
+    from backend side we get common name of placeholder is called 'nameOfContainer-{*}' where '{*}' marker for replacing **/
+    const isExistsDynamicPlaceholders =
+      rendering?.placeholders &&
+      Object.keys(rendering.placeholders).find((placeholder) => placeholder.indexOf('{*}') !== -1);
+    if (isExistsDynamicPlaceholders) {
+      Object.keys(rendering.placeholders).forEach((placeholder) => {
+        if (placeholder.indexOf('container-{*}') !== -1) {
```

Comment:
I'm confused about the expected placeholder name format. In the `isExistsDynamicPlaceholders` we are checking for `{*}` but here we are using `container-{*}`. Is "container-" always present or not? If so, we could keep the constant, but rename it as postfix e.g. `DYNAMIC_PLACEHOLDER_POSTFIX = 'container-{*}';`

Reply:
@ambrauer My bad. This `container-{*}` was for testing and old code. I've fixed it